hudi-utilities/src/main/java/org/apache/hudi/utilities/multitable/MultiTableServiceUtils.java

```diff
@@ -44,22 +46,43 @@
  * Utils for executing multi-table services.
  */
 public class MultiTableServiceUtils {
+  private static final Logger LOG = LoggerFactory.getLogger(MultiTableServiceUtils.class);
 
   public static class Constants {
     public static final String TABLES_TO_BE_SERVED_PROP = "hoodie.tableservice.tablesToServe";
 
+    public static final String TABLES_SKIP_WRONG_PATH = "hoodie.tableservice.skip.wrong.path";
```

Comment:
rename to `hoodie.tableservice.skipNonHudiTable` would be better ?

Reply:
sure